products/logging/api.yaml

```diff
@@ -28,7 +28,7 @@ objects:
   - !ruby/object:Api::Resource
     name: "Metric"
     base_url: projects/{{project}}/metrics
-    self_link: "projects/{{project}}/metrics/{{%name}}"
```

Comment:
This is actually correct! We added `%` here to indicate "URL-encode this value", and added support for doing that in TF. Looking at blame will probably give you enough context for Ansible.

Reply:
Sounds good. I'll make that change + document the % thing.